src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -2362,6 +2362,67 @@ public void TestRuntimeConfigSetupWithNonJsonConstructor()
                 $"Config does not have a cosmos datasource and member {nameof(runtimeConfig.CosmosDataSourceUsed)} must be marked as false.");
         }
 
+        /// <summary>
+        /// Test to validate pagination options.
+        /// </summary>
+        /// <param name="exceptionExpected">Should there be an exception.</param>
+        /// <param name="defaultPageSize">default page size to go into config.</param>
+        /// <param name="maxPageSize">max page size to go into config.</param>
+        /// <param name="expectedExceptionMessage">expected exception message in case there is exception.</param>
+        /// <param name="expectedDefaultPageSize">expected default page size from config.</param>
+        /// <param name="expectedMaxPageSize">expected max page size from config.</param>
+        [DataTestMethod]
+        [DataRow(false, null, null, "", (int)PaginationOptions.DEFAULT_PAGE_SIZE, (int)PaginationOptions.MAX_PAGE_SIZE,
+            DisplayName = "MaxPageSize should be 100,000 and DefaultPageSize should be 100 when no value provided in config.")]
+        [DataRow(false, 1000, 10000, "", 1000, 10000,
+            DisplayName = "Valid inputs of MaxPageSize and DefaultPageSize must be accepted and set in the config.")]
+        [DataRow(false, -1, 10000, "", 10000, 10000,
+            DisplayName = "DefaultPageSize should be the same as MaxPageSize when input is -1")]
+        [DataRow(false, 100, -1, "", 100, Int32.MaxValue,
+            DisplayName = "MaxPageSize should be UInt32.MaxValue when input is -1")]
```

Comment:
need to be specific about what field you are referring to when you say "input". The two displaynames for these tests aren't consistent because "input" may be referring to `defaultPageSize` or `maxPageSize`

Reply:
Oh i meant to refer to both here. this is the valid scenario where both are correct. Hence valid inputs of both should be accepted in config.